db-async-common/src/main/java/com/github/jasync/sql/db/column/InetAddressEncoderDecoder.kt

```diff
@@ -1,20 +1,8 @@
 package com.github.jasync.sql.db.column
 
 import java.net.InetAddress
-import sun.net.util.IPAddressUtil.textToNumericFormatV4
-import sun.net.util.IPAddressUtil.textToNumericFormatV6
 
 object InetAddressEncoderDecoder : ColumnEncoderDecoder {
-
-    override fun decode(value: String): Any {
-        return if (value.contains(':')) {
-            InetAddress.getByAddress(textToNumericFormatV6(value))
-        } else {
-            InetAddress.getByAddress(textToNumericFormatV4(value))
-        }
-    }
-
-    override fun encode(value: Any): String {
-        return (value as InetAddress).hostAddress
-    }
+    override fun decode(value: String): Any = InetAddress.getByName(value)
+    override fun encode(value: Any): String = (value as InetAddress).hostAddress
```

Comment:
textToNumericFormatV4/V6 appear to be inaccessible. I'm not sure why this wasn't building on my setup, but I figured it was probably best to migrate off of references to the sun.* package.